lib/core/models/discovery.dart

```diff
@@ -0,0 +1,10 @@
+// FLUTTER / DART / THIRD-PARTIES
+import 'package:flutter/material.dart';
+
+class Discovery {
```

Comment:
Maybe adding a potential image or animation? :thinking:

Reply:
Yeah that's a thing that had me worried about the structure right now. But, I just tought about something, what about we put a simple widget reference that will be used to construct the detail page (below the title). So in the files discovery_ids.dart (that we could probably rename now), we would have for each different discovery a way to build them.

For example, with the new version of @camillebrulotte in [add_discovery_option_in_settings](https://github.com/ApplETS/Notre-Dame/blob/feature/discovery%2Fadd_discovery_option_in_settings/lib/core/constants/discovery_ids.dart), she put a list of strings that will be useful to build the details widget as such:
```dart
Discovery(
  path: RouterPaths.dashboard,
  featureId: 'navbar_dashboard_page_id',
  title: intl.discovery_navbar_dashboard_page_title,
  details: [
    intl.discovery_navbar_dashboard_page_details,
    intl.discovery_navbar_dashboard_page_details2
  ]),
[...]
```

Instead of defining the widget variables at 2 different places, like we do, maybe we could simply add the widget (detail) to construct it directly in the list? Example:
  ```dart
Discovery(
    path: RouterPaths.dashboard,
    featureId: 'navbar_dashboard_page_id',
    title: intl.discovery_navbar_dashboard_page_title,
    detail: Column(
        crossAxisAlignment: CrossAxisAlignment.start,
        children: <Widget>[
          Text(intl.discovery_navbar_dashboard_page_details, textAlign: TextAlign.justify),
          RichText(
            textAlign: TextAlign.justify,
            text: TextSpan(
              children: [
                const TextSpan(text: '\n'),
                const WidgetSpan(child: Icon(Icons.restore, size: 16, color: Colors.white)),
                TextSpan(text: ' ${intl.discovery_navbar_dashboard_page_details2}'),
              ],
            ),
          ),
        ],
      ),
    ),
[...]
```

So that way we can directly put an icon, or an animation without too much hassle in a discovery feature. What do you think?

Reply:
I think this is a great idea! (Sorry for the delay :sweat_smile: ) Go with it :)

Reply:
Alright, I'll be on it! And no problem! 😄 I also forgot to bring it up last meeting.